components/brave_new_tab_ui/components/default/braveToday/index.tsx

```diff
@@ -51,6 +50,7 @@ const intersectionOptions = { root: null, rootMargin: '0px', threshold: 0.25 }
 
 export default function BraveTodaySection (props: Props) {
   const dispatch = useDispatch()
+  const [optedIn, setOptedIn] = useNewTabPref('isBraveTodayOptedIn')
```

Comment:
Nice work!

Reply:
Yeah, I made the `useNewTabPref` hook a while back, and I've been gradually migrating all our prop drilling over to it :laughing: It's definitely a fair bit less code.